lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1,14 +1,25 @@
 {
+  "fetchTime": "2019-07-23T16:41:05.852Z",
```

Comment:
I don't think most of these are necessary @khempenius , correct?

Could you revert this file and rerun with just the name of the artifact you want to update?

```
yarn update:sample-artifacts settings
```

for example